modules/mapping-utils/src/resolveHits.js

```diff
@@ -98,7 +98,13 @@ export default type => async (
     let source = x._source;
     let nested_nodes = resolveNested({ node: source, nestedFields });
     return {
-      searchAfter: x.sort || [],
+      searchAfter:
+        x.sort?.map(
+          x =>
+            Number.isInteger(x) && !Number.isSafeInteger(x)
+              ? `-9223372036854775808`
```

Comment:
it could be worth making a constant `ES_MAX_INT`, partly so we can reference it again if needed but also just to help future consumers of the code with the semantic :P

Reply:
yeah good call... I was just lazy and didn't want to have to deal w/ the cross-dep between server & middleware to get those constants here lol

Reply:
might also be good to add a `// TODO: ` with a link to that issue :)

Reply:
👋  yeahhh good call